src/resolvers/federationSources.ts

```diff
@@ -10,7 +10,8 @@ export default function federationSources(): {
   return {
     orphanedTypes: [],
     referenceResolvers: {
-      // $Name: { __resolveReference: resolve$NameReference },
+      // Example:
```

Comment:
Not sure if this comment is needed at all or you want to keep it.

Reply:
I think it is nice to keep the comment. Might become handy.